review.md

```diff
@@ -0,0 +1,92 @@
+# Code Review principles and tips
+
+## Why do we code review?
+
+1. Code reviews are essential to improve the level of our code, and to prevent bugs. 
+
+2. A great and fast way for both the reviewer and the code owner to share ideas,  
+   learn, and improve their code writing and understanding skills.
+
+3. Allow us as reviewers to get to know parts of the code base we usually  
+   don't work on.
+
+4. Help our fellow team members to validate their code quality and push updates faster.
+
+## Tips for reviewing
+
+-   It is important to maintain a healthy, respectful, and civil environment. Feel free  
+    to ask questions, offer suggestions, or provide criticism, regardless of your  
+    position or level of expertise. However, it is crucial to avoid personal attacks  
+    and focus on reviewing the code rather than criticizing the individual who wrote it.  
+    Additionally, please refrain from using capital letters (all caps), using  
+    inappropriate language, or addressing others directly.  
+    Here are a few examples:
```

Comment:
I suggest remove the spaces at end of line. The idea being that we control the line length for the code, which we let the rendered text to be as long as the screen it is read from allows.
```suggestion
-   It is important to maintain a healthy, respectful, and civil environment. Feel free
    to ask questions, offer suggestions, or provide criticism, regardless of your
    position or level of expertise. However, it is crucial to avoid personal attacks
    and focus on reviewing the code rather than criticizing the individual who wrote it.
    Additionally, please refrain from using capital letters (all caps), using
    inappropriate language, or addressing others directly.
    Here are a few examples:
```

Reply:
fixed